charts/shoot-core/components/charts/konnectivity-agent/templates/daemonset.yaml

```diff
@@ -0,0 +1,69 @@
+{{- if semverCompare ">= 1.18" .Values.kubernetesVersion }}
+apiVersion: {{ include "daemonsetversion" . }}
+kind: DaemonSet
```

Comment:
Why is this a `DaemonSet`? A `Deployment` would also be sufficient?

Reply:
load distribution basically, the server picks a random backend. Also, if for some reason the node is silently not healthy, it would be nice to still have a working connection to the shoot.